examples/header_bar/main.rs

```diff
@@ -0,0 +1,46 @@
+mod title_bar;
+
+use gtk::gio;
+use gtk::prelude::*;
+use gtk::CheckButton;
+use gtk::Label;
+use gtk::Stack;
+use gtk::{ApplicationWindow, WindowPosition};
```

Comment:
```suggestion
use gtk::{ApplicationWindow, CheckButton, Label, Stack, WindowPosition};
```

Reply:
Is it not necessary to include prelude?

```rust
use gtk::prelude::*;
use gtk::{ApplicationWindow, CheckButton, Label, Stack, WindowPosition};
```
vs

```rust
use ;
use gtk::{gtk::prelude::*, ApplicationWindow, CheckButton, Label, Stack, WindowPosition};
```

Reply:
my code suggestion doesn't remove the prelude, it is required yes.

Reply:
Modified in the title bar and main!